src/core/screen.js

```diff
@@ -45,6 +45,7 @@ export class Screen extends Phaser.Scene {
             activeScreens,
             navigation: this._data.navigation,
             transientData: this._data.transient || {},
+            registry: this._data.registry,
```

Comment:
As discussed - this isn't required. In generally we also avoid sharing anything as much as possible via classes what with classes being terrible - unfortunately Phaser being class based makes this somewhat difficult to avoid.

Reply:
Yeah, meant to take this out. TY =)

Reply:
672e4a9